tests/phpunit/tests/l10n/wpWordCount.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/**
+ * @group l10n
+ * @group i18n
+ *
+ * @covers ::wp_word_count
+ */
+class Tests_L10n_wpWordcount extends WP_UnitTestCase {
+	/**
+	 * Tests that words are counted correctly based on the type.
+	 *
+	 * @ticket 56698
+	 *
+	 * @dataProvider data_get_string_variations
+	 *
+	 * @param string $string                      String to count words.
```

Comment:
```suggestion
	 * @param string $text                        Text to count elements in.
```
We avoid using data types as variable names. Let's switch `$string` to `$text` to match the function under test.